backend/src/services/release.ts

```diff
@@ -276,11 +291,36 @@ export async function getReleasesForExport(user: UserInterface, modelId: string,
   return releases
 }
 
+export function semverStringToObject(semver: string): SemverObject {
+  const vIdentifierIndex = semver.indexOf('v')
+  const trimmedSemver = vIdentifierIndex === -1 ? semver : semver.slice(vIdentifierIndex + 1)
+  const [version, metadata] = trimmedSemver.split('-')
+  const [major, minor, patch] = version.split('.')
+  const majorNum: number = Number(major)
+  const minorNum: number = Number(minor)
+  const patchNum: number = Number(patch)
+  return { major: majorNum, minor: minorNum, patch: patchNum, ...(metadata && { metadata }) }
+}
+
+export function semverObjectToString(semver: SemverObject): string {
+  if (!semver) {
+    return ''
+  }
+  let metadata: string
```

Comment:
This type isn't strictly true as it's `undefined` until you assign a value. Instead I'd suggest `let metadata = ''`

Reply:
Changed